cli/ops.rs

```diff
@@ -200,6 +200,9 @@ pub fn op_selector_std(inner_type: msg::Any) -> Option<CliDispatchFn> {
     msg::Any::CreateWorker => Some(op_create_worker),
     msg::Any::Cwd => Some(op_cwd),
     msg::Any::Dial => Some(op_dial),
+    msg::Any::DlOpen => Some(op_dl_open),
+    msg::Any::DlSym => Some(op_dl_sym),
+    msg::Any::DlCall => Some(op_dl_call),
```

Comment:
Maybe these names are confusing because it's not a true dlopen/dlsym binding.

(I realize I asked for these renames initially. Sorry.)

I suggest op_plugin_open, op_plugin_sym, op_plugin_call. What do you think?

Reply:
👍